script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -78,33 +78,33 @@ Good morning, Arihiko. Are you finally starting to care about your attendance?"
 "No, I'm not worried about that. There's still plenty of room for me to skip.# I've got my sister at home today, so I just can't slack off.
 Besides, I've been staying up all night and I'm at the end of my rope."
 "I see. My bad, let me redo my greeting.
-Good night, Asshiko. Hurry up to your desk and get some sleep."
+Good night, Assihiko. Hurry up to your desk and get some sleep."
 <Page21>
-"Gee, what a cold reaction.
-That's your attitude toward your rival from middle school? Can't you be more passionate about it!?"
+"Come on, what a cold reaction.
+That's your attitude toward your rival from middle school? Couldn't you be more passionate about it!?"
 <Page22>
 "...Why do you and I have to be rivals?
 If you want to live an unprincipled life, find some like-minded people on the street. Don't bother me too much."
 No, I've lent him almost 10,000 yen since middle school and he hasn't given it back, so I can call him a foreign enemy at this point.
 <Page23>
-"I'm not sure why you're so cold only to me.
+"I'm not sure why you're so cold only to me. //So cold only to me or only so cold to me?
 You're like a saint to everyone else, isn't it unfair?"
 "What, didn't you already know?
 There's nothing fair about the world."
 <Page24>
-"That's true, but...
+"That's true...
 But aren't you especially cold to me?"
 <Page25>
 Arihiko sighs exaggeratedly.
-It isn't that I am being consciously cold to him, it is just that we have naturally come this close as a result of our mutual frankness without hesitation.
+It isn't that I am being consciously cold to him, it's just that we have naturally come this close as a result of our mutual frankness without hesitation. //Rephrase needed?
```

Comment:
> 意識して冷たくあたっている訳ではなくて、互いに気兼ねなく本音を言い合っている結果、自然にこういった距離感になっているだけだった。

Reply:
It's not like I'm being cold to him on purpose, it's just a natural consequence of how frank we are with each other. ?

Reply:
it's just that we have naturally come this close as a result of how frank we are with each other.